openslides_backend/models/models.py

```diff
@@ -202,6 +204,59 @@ class OrganizationTag(Model):
     )
 
 
+class Theme(Model):
+    collection = Collection("theme")
+    verbose_name = "theme"
+
+    id = fields.IntegerField(required=True)
+    name = fields.CharField(required=True)
+    accent_100 = fields.ColorField()
+    accent_200 = fields.ColorField()
+    accent_300 = fields.ColorField()
+    accent_400 = fields.ColorField()
+    accent_50 = fields.ColorField()
+    accent_500 = fields.ColorField(required=True)
+    accent_600 = fields.ColorField()
+    accent_700 = fields.ColorField()
+    accent_800 = fields.ColorField()
+    accent_900 = fields.ColorField()
+    accent_A100 = fields.ColorField()
+    accent_A200 = fields.ColorField()
+    accent_A400 = fields.ColorField()
+    accent_A700 = fields.ColorField()
+    primary_100 = fields.ColorField()
+    primary_200 = fields.ColorField()
+    primary_300 = fields.ColorField()
+    primary_400 = fields.ColorField()
+    primary_50 = fields.ColorField()
+    primary_500 = fields.ColorField(required=True)
+    primary_600 = fields.ColorField()
+    primary_700 = fields.ColorField()
+    primary_800 = fields.ColorField()
+    primary_900 = fields.ColorField()
+    primary_A100 = fields.ColorField()
+    primary_A200 = fields.ColorField()
+    primary_A400 = fields.ColorField()
+    primary_A700 = fields.ColorField()
+    warn_100 = fields.ColorField()
+    warn_200 = fields.ColorField()
+    warn_300 = fields.ColorField()
+    warn_400 = fields.ColorField()
+    warn_50 = fields.ColorField()
+    warn_500 = fields.ColorField(required=True)
+    warn_600 = fields.ColorField()
+    warn_700 = fields.ColorField()
+    warn_800 = fields.ColorField()
+    warn_900 = fields.ColorField()
+    warn_A100 = fields.ColorField()
+    warn_A200 = fields.ColorField()
+    warn_A400 = fields.ColorField()
+    warn_A700 = fields.ColorField()
+    organization_ids = fields.RelationListField(
```

Comment:
Why is this a list field? I think a simple `RelationField` suffices here, since there can only ever be one organisation